### ISSUE TITLE
Sober Not Opening - Empty Latest.Log

### ISSUE THREAD

**Issue by unixpill** [vinegarhq/sober#502]
### Acknowledgement of checking existent issues

- [x] I have checked if my issue has already been reported, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

Due to unknown means, I first ran sober to which it automatically updated. It then, upon the next run, failed to do anything. It printed nothing per run with ps -A resulting in seeing flatpak-session. 

I went to check latest.log in which it had nothing. It was a 0b file. If there's anyone with a similar issue, I'd love to hear.

I'm on X11.

**Comment by tunis4** [comment]
send output of `sudo dmesg | grep sober`

**Comment by jcvm-dev** [comment]
Empty

**Comment by unixpill** [comment]
[ 4827.421772] sober[20020]: segfault at ffffffffffbffffe ip 00007f194998548d sp 00007ffea2a1fb98 error 5 in libc.so.6[7f1949829000+175000] likely on CPU 6 (core 6, socket 0)
[ 4940.373627] sober[20699]: segfault at ffffffffffbffffe ip 00007f014518548d sp 00007ffc59f7d308 error 5 in libc.so.6[7f0145029000+175000] likely on CPU 5 (core 5, socket 0)
[ 5015.877037] sober[20785]: segfault at ffffffffffbffffe ip 00007f873ae9648d sp 00007ffd78bd89a8 error 5 in libc.so.6[7f873ad3a000+175000] likely on CPU 4 (core 4, socket 0)
[ 5101.957389] sober[20935]: segfault at ffffffffffbffffe ip 00007fb40518548d sp 00007ffe8e855de8 error 5 in libc.so.6[7fb405029000+175000] likely on CPU 5 (core 5, socket 0)
[ 5355.981638] sober[23407]: segfault at ffffffffffbffffe ip 00007f4a3dc9648d sp 00007ffd0cf96538 error 5 in libc.so.6[7f4a3db3a000+175000] likely on CPU 5 (core 5, socket 0)
[ 5440.975811] sober[23796]: segfault at ffffffffffbffffe ip 00007fe53049648d sp 00007ffe1497da28 error 5 in libc.so.6[7fe53033a000+175000] likely on CPU 5 (core 5, socket 0)
[ 5467.634655] sober[24005]: segfault at ffffffffffbffffe ip 00007f205318548d sp 00007fff2cbf86f8 error 5 in libc.so.6[7f2053029000+175000] likely on CPU 5 (core 5, socket 0)
[ 5679.763455] sober[24887]: segfault at ffffffffffbffffe ip 00007fa174b8548d sp 00007fff2f299b88 error 5 likely on CPU 5 (core 5, socket 0)
[ 9910.801572] sober[39507]: segfault at ffffffffffbffffe ip 00007f7cb7e9648d sp 00007ffecf434488 error 5 in libc.so.6[7f7cb7d3a000+175000] likely on CPU 5 (core 5, socket 0)
[10254.232372] sober[40857]: segfault at ffffffffffbffffe ip 00007f524af8548d sp 00007ffed14bf4b8 error 5 in libc.so.6[7f524ae29000+175000] likely on CPU 5 (core 5, socket 0)

> send output of `sudo dmesg | grep sober`

**Comment by unixpill** [comment]
Note: tried both autoinstall and manual install

**Comment by tunis4** [comment]
can you send output of neofetch

**Comment by unixpill** [comment]
Kernel: x86_64 Linux 6.1.0-28-amd64
OS: Debian 12
Shell: bash 5.2.15
CPU: ryzen 9 5900x 
RAM: 32 GB DDR5
GPU: RTX 3090

That should be good enough. I config'd my neofetch a ton and honestly dont wanna have to download a new one.

**Comment by Xless-Qu-dev** [comment]
Same thing happening here. Worked for a few days after reinstalling but is now broken again.

**Comment by unixpill** [comment]
What's your OS? I might try in a bit on one of my other machines to test if this is a Deb issue.

**Comment by unixpill** [comment]
gdb:
```
Starting program: /usr/bin/flatpak run org.vinegarhq.Sober
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[New Thread 0x7ffff35ff6c0 (LWP 5043)]
[New Thread 0x7ffff2dfe6c0 (LWP 5044)]
[Detaching after fork from child process 5048]
[Thread 0x7ffff2dfe6c0 (LWP 5044) exited]
[Thread 0x7ffff35ff6c0 (LWP 5043) exited]
warning: opening /proc/PID/mem file for lwp 5040.5040 failed: Permission denied (13)
process 5040 is executing new program: /proc/5040/exe
/proc/5040/ns/mnt: Permission denied.
```

**Comment by kxtbit** [comment]
I appear to be having the same issue. When doing `flatpak run org.vinegarhq.Sober` in the terminal, I just get:
```
ERROR: Failed to load resource: https://sober.vinegarhq.org/app
ERROR: Failed to load resource: https://setup.rbxcdn.com/version-8aa36bbf0eb1494a-content-textures3.zip

FATAL: Couldn't reach Roblox to download assets
```
A "Sober Crash" window also appears with the same message "Couldn't reach Roblox to download assets".
My output of `sudo dmesg | grep sober` is:
```
[94007.503244] sober[318824]: segfault at ffffffffffbffffe ip 00007331a2e9f54d sp 00007fffc7515cd8 error 5 in libc.so.6[18d54d,7331a2d3a000+175000] likely on CPU 15 (core 3, socket 0)
[94019.579769] sober[319358]: segfault at ffffffffffbffffe ip 00007360d318e54d sp 00007ffddffc72e8 error 5 in libc.so.6[18d54d,7360d3029000+175000] likely on CPU 11 (core 13, socket 0)
[94031.262933] sober[319430]: segfault at ffffffffffbffffe ip 00007108c929f54d sp 00007fff6e8b2cf8 error 5 in libc.so.6[18d54d,7108c913a000+175000] likely on CPU 15 (core 3, socket 0)
[94304.685505] sober[320194]: segfault at ffffffffffbffffe ip 00007a0e6b98e54d sp 00007ffe72d9b0c8 error 5 in libc.so.6[18d54d,7a0e6b829000+175000] likely on CPU 5 (core 5, socket 0)
[95624.902339] sober[342366]: segfault at ffffffffffbffffe ip 0000749fe509f54d sp 00007ffc85680f28 error 5 in libc.so.6[18d54d,749fe4f3a000+175000] likely on CPU 15 (core 3, socket 0)
[96416.861182] sober[344736]: segfault at ffffffffffbffffe ip 000078d85969f54d sp 00007ffe7b510948 error 5 in libc.so.6[18d54d,78d85953a000+175000] likely on CPU 3 (core 3, socket 0)
[96609.592541] sober[345935]: segfault at ffffffffffbffffe ip 00007c04bac9f54d sp 00007ffec11a0478 error 5 in libc.so.6[18d54d,7c04bab3a000+175000] likely on CPU 17 (core 5, socket 0)
[97033.169004] sober[347587]: segfault at ffffffffffbffffe ip 000071b7fa78e54d sp 00007ffc6bc0b088 error 5 in libc.so.6[18d54d,71b7fa629000+175000] likely on CPU 22 (core 12, socket 0)
[97462.216254] sober[348674]: segfault at ffffffffffbffffe ip 00007d2fdcb8e54d sp 00007ffc5028d878 error 5 in libc.so.6[18d54d,7d2fdca29000+175000] likely on CPU 15 (core 3, socket 0)
[97469.092288] sober[348773]: segfault at ffffffffffbffffe ip 000079ebddf8e54d sp 00007ffe65c0ddf8 error 5 in libc.so.6[18d54d,79ebdde29000+175000] likely on CPU 17 (core 5, socket 0)
```
I believe each of those lines is for one time I tried to run Sober, since every time I run it a new segfault appears in dmesg.

**Comment by unixpill** [comment]
Yeah this looks highly similar to mine. You got output though so that's good. Can you try running with GDB?
